### DISCUSSION TITLE
Displaying a number as currency

### DISCUSSION THREAD

**Discussion by DavidDMuir** [sqlitebrowser/sqlitebrowser#3221] [Q&A] [answered]
Is it possible to display numbers in the Browse data tab as currency? For example, if the contents of the field is: 12.5, can you change the display format so it shows as: £12.50?

**Comment by MKleusberg** [comment, marked as answer]
You can use a custom display format for that. To do so right-click the column header and click "Edit display format". In the dialog select "Custom" and type in the following format where "column_name" is the name of the selected column:
```sql
format('£%.2f', "column_name")
```

**Reply by chrisjlocke** [reply]
Or printf if sqlite moans about format not being a valid function.

![image](https://user-images.githubusercontent.com/7872218/209478318-d6e9f9e9-3c3d-47b8-a781-bada85d63622.png)

**Reply by bmorgy** [reply]
@chrisjlocke - why does DB Browser not recognize the **format** command?  I see online that **printf** was superseded from v.3.8.  How can I fix this?  I prefer **format** in my python code but always need to replace them when using/testing in DB Browser.

**Reply by chrisjlocke** [reply]
I don't think its in the 3.12.2 version of sqlite, as that uses an older version of sqlite.
